### DISCUSSION TITLE
yt-dlp script

### DISCUSSION THREAD

**Discussion by solsticedhiver** [jcorporation/myMPD#1228] [Q&A] [answered]
Currently I am copying an url from my browser in a terminal and run:
```
yt-dlp -g theurl
```
then copy that new url, and feed to mpd via mympd.

Is there a way (a script, a plugin) to automate that at mympd level to onyl enter the url and have mympd use yt-dlp to get the real url?

**Comment by jcorporation** [comment]
You can write a lua script.

**Comment by jcorporation** [comment, marked as answer]
#1276
